keepassxc-browser/keepassxc-browser.js

```diff
@@ -1559,7 +1559,7 @@ cip.initCredentialFields = function(forceCall) {
         cip.initializeSitePreferences();
         if (cip.settings.sitePreferences) {
             for (const site of cip.settings.sitePreferences) {
-                if (site.url === document.location.href || siteMatch(site.url, document.location.href)) {
+                if (site.url === document.location.href || siteMatch(site.url, window.top.location.href)) {
```

Comment:
The only time ```document.location.href``` and ```window.top.location.href``` would be different is in a frames scenario. In this case, is this the desired behavior? You would also want to change the other half of the if statement if it is.

Reply:
Yes, this is indeed the required behavior. #165 handles about wysiwyg editors who add additional data-cip-id attributes to input fields. They use iframes which firefox sees as a frame with a different document.location (which is actually a empty string) Chrome/chromium has different behavior and was already solved. This fix does not change anything to chromium.

Reply:
Great!